### ISSUE TITLE
[Suggestion] Textures License

### ISSUE THREAD

**Issue by JurajVajda** [quiqueck/BetterNether#110]
### Description

While the license says that the mod is MIT, the textures don't fall under the same license. It would be helpful to have list of textures and their licenses included in LICENSE file (or somewhere in the repo) so its clear what textures are included under public domain and what textures are restricted or non-reusable.

**Comment by quiqueck** [comment]
Hi, and thanks for the suggestion. Did you see this one:
https://github.com/quiqueck/BetterNether/blob/1.19/LICENSE.ASSETS

**Comment by JurajVajda** [comment]
@quiqueck yes I have seen it, it seem like that file is missing these textures:
https://github.com/quiqueck/BetterNether/tree/1.19/src/main/resources/assets/betternether/textures
